modules/account-lib/src/coin/sol/transactionBuilder.ts

```diff
@@ -15,7 +15,6 @@ import { instructionParamsFactory } from './instructionParamsFactory';
 
 export abstract class TransactionBuilder extends BaseTransactionBuilder {
   private _transaction: Transaction;
-  private _signatures: Interface.Signature[] = [];
```

Comment:
Updated the access modifier, might need Joey for the solana review

Reply:
Actually we can't really make changes to move this field to the base transaction builder. Some other coins already have different implementations of how they treat their signatures:

![image](https://user-images.githubusercontent.com/25713361/154590028-ebd9a9e9-bc2c-44a3-b746-541108f2782a.png)

Will need to remove this

Reply:
got it. dropped the commit. We can access the field using `this._signatures`